src/css/layout.css

```diff
@@ -80,10 +80,10 @@
 }
 
 .cursor-coordinates {
-  color:#888;
-  font-size:12px;
-  font-weight:bold;
-  font-family:monospace;
+  color: gold;
```

Comment:
Can we use var(--highlight-color) for gold? (defined in variables.css)

Reply:
Sorry, didn't notice this till now. Of course. No problem.